fixtures/additional-modules/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import dep from "./dep";
+import text from "./text.txt";
+
+export default <ExportedHandler>{
+	async fetch(request) {
+		const url = new URL(request.url);
+		if (url.pathname === "/dep") {
+			return new Response(dep);
+		}
+		if (url.pathname === "/text") {
+			return new Response(text);
+		}
+		if (url.pathname === "/dynamic") {
+			return new Response((await import("./dynamic.js")).default);
```

Comment:
```suggestion
			const identifier = "./dynamic.js";
			return new Response((await import(identifier)).default);
```

Think you missed this other one :)

Reply:
I didn't bother with that one.
We only need one to catch if esbuild starts to bundle stuff that we wanted to be external.
Or am I missing something significant here?